src/main/java/Duke.java

```diff
@@ -1,10 +1,159 @@
+                System.out.println("Nice! I've marked this task as done:\n"+
+                       "["+tasks[indexing-1].getStatusIcon()+"] "+ list[indexing-1]);
+                System.out.println("____________________________________________________________\n");
+
+            }
+
+            else if(line.toLowerCase().contains("todo") || line.toLowerCase().contains("deadline") || line.toLowerCase().contains("event") )
+            {
+
+
+                if(line.toLowerCase().contains("todo"))
+                {
+                    String TodoTask =line.toLowerCase().substring(5,line.length());
+                    list[i]=TodoTask;
+                    tasks[i] = new Todo(TodoTask);
+
+                }
+                else if(line.toLowerCase().contains("deadline")) {
+                    String[] ToSplitDeadline=line.split("/");
+                    String DeadlineTask =ToSplitDeadline[0].toLowerCase().substring(9,ToSplitDeadline[0].length()-1);
+                    list[i]=DeadlineTask;
+                    tasks[i] = new Deadline(DeadlineTask, ToSplitDeadline[1].substring(3,ToSplitDeadline[1].length()));
+                }
+                else if(line.toLowerCase().contains("event"))
+                {
+                    String[] ToSplitEvent=line.split("/");
+                    String EventTask =ToSplitEvent[0].toLowerCase().substring(6,ToSplitEvent[0].length());
+                    list[i]=EventTask;
+                    tasks[i] = new Event(EventTask, ToSplitEvent[1].substring(5,ToSplitEvent[1].length()),ToSplitEvent[2].substring(3,ToSplitEvent[2].length()));
+
+                }
+                skip=true;
+
+                System.out.println("____________________________________________________________");
+                System.out.println("Got it. I've added this task:\n"+
+                                "  "+ tasks[i]+ "\n"+
+                        "Now you have "+ (i+1) +" tasks in the list.");
+
+                System.out.println("____________________________________________________________\n");
+                i+=1;
+
+
+            }
+
+
+            if(line.equalsIgnoreCase("list"))
+            {
+                skip=true;
+                System.out.println("____________________________________________________________\n"+
+                        "Here are the tasks in your list:");
+
+                for(int m=0;m<i;m+=1)
+                {
+                    int index=m+1;
+                    if(tasks[m]==null)
+                    {
+                        System.out.println(index + "." +"["+ t.getStatusIcon()+ "] "+list[m]);
+
+                    }
+                    else {
+                        //the [X] or []
+                        System.out.println(index + "." + tasks[m]);
+                    }
+
+                }
+                System.out.println( "____________________________________________________________");
+
+            }
+            else {
+                if(!skip) {
```

Comment:
Is it possible to avoid deep nesting here? The code seems to be of arrowhead style.